StreamingAnalytics/LowLatencySample/Temperature-Epl/monitors/TemperatureEventMonitor.mon

```diff
@@ -0,0 +1,29 @@
+package lowlatency;
+using com.apama.aggregates.mean;
+/** If average temperature in 5 minutes is greater than 80 C, send a shutdown alert  */
+
+event Temperature {
+    float temperature;
+}
+
+event ShutDown {
+    string shutdownMessage;
+}
+
+monitor TempEventMonitor {
+    constant float FIVE_MINS_INTERVAL := 300.0;
+    constant float THRESHOLD_TEMPERATURE := 80.0;
+    action onload() {
```

Comment:
I think putting an empty line above this is better for readability.

Reply:
Done